extras/check_version.sh

```diff
@@ -25,4 +26,14 @@ if [[ x${PACKAGE_VERSION}x != x${OPENAPI_VERSION}x ]]; then
 	EXITCODE=-1
 fi
 
+if [[ -f "$BUILD_VERSION_FILE" ]]; then
+	# Get the build version and ignore the suffix
+    BUILD_VERSION=$(cat "$BUILD_VERSION_FILE" | cut -d"-" -f1)
+
+	if [[ x${PACKAGE_VERSION}x != x${BUILD_VERSION}x ]]; then
+		echo "Version different in ${PACKAGE_FILE} and ${BUILD_VERSION_FILE}"
```

Comment:
The BUILD_VERSION shoudl'nt always be equal to the other versions. This may fail sometimes.